### DISCUSSION TITLE
Error loading xtrack.line / .json file

### DISCUSSION THREAD

**Discussion by BhawinDhital** [xsuite/xsuite#265] [Q&A] [answered]
Hello,

I am very new to xsuite. I am trying to use my MAD-X lattice using xtrack.line and mad.sequence[] in xsuite. It gives me errors: 
**ValueError: Element None not supported,
 implement add_None or convert_None in function in MadLoader**

**ValueError: Element sextupole not supported,
 implement add_sextupole or convert_sextupole in function in MadLoader**

Also, I am unable to generate .json file from my lattice. How do I get a .json file in xsuite?
<img width="1130" alt="Scree
<img width="1144" alt="Screen Shot 2022-12-15 at 7 14 06 PM" src="https://user-images.githubusercontent.com/46133920/208173764-4dcdbb43-7f8c-4f8a-8ef6-2cea1e60d7e5.png">
n Shot 2022-12-15 at 7 13 30 PM" src="https://user-images.githubusercontent.com/46133920/208173752-c814af82-ea5e-497b-8b04-34f509d05555.png">

Thank you,
Bhawin Dhital
BNL, USA

**Comment by giadarol** [comment, marked as answer]
Hello, so far we have supported only thin elements. You can use the makethin command of mad-x to transform your lattice to include only thin elements. An example showing how to do that is available here (lines 35-47) https://github.com/xsuite/xtrack/blob/d5d5d90303a3deabed197327033d979a2bf030e3/examples/tapering/100_generate_fcc_line_no_rad.py#L35
